### ISSUE TITLE
使用onnxruntime对glm_block_0.onnx进行推理报错BroadcastIterator::Init(ptrdiff_t, ptrdiff_t) axis == 1 || axis == largest was false. Attempting to broadcast an axis by a dimension other than 1. 4 by 32

### ISSUE THREAD

**Issue by guohuanliang1** [wangzhaode/mnn-llm#90]
我下载了onnx模型，使用如下代码进行推理报错：
  File "/root/anaconda3/envs/xxx/lib/python3.9/site-packages/onnxruntime/capi/onnxruntime_inference_collection.py", line 217, in run
    return self._sess.run(output_names, input_feed, run_options)
onnxruntime.capi.onnxruntime_pybind11_state.RuntimeException: [ONNXRuntimeError] : 6 : RUNTIME_EXCEPTION : Non-zero status code returned while running Where node. Name:'Where_284' Status Message: /onnxruntime_src/onnxruntime/core/providers/cpu/math/element_wise_ops.h:540 void onnxruntime::BroadcastIterator::Init(ptrdiff_t, ptrdiff_t) axis == 1 || axis == largest was false. Attempting to broadcast an axis by a dimension other than 1. 4 by 32

import onnxruntime as ort
import numpy as np

batch_size = 1
seq_len = 20
inputs_embeds = np.empty((1, 1, 4096), dtype = np.float32)
attention_mask = np.array([[[[False, False, False,  True],
          [False, False, False,  True],
          [False, False, False,  True],
          [False, False, False, False]]]])
position_ids = np.array(([[[0, 1, 2, 2], [0, 0, 0, 1]]]))
past_key_values = np.empty((2, 28, 1, 32, 128), dtype = np.float32)

session = ort.InferenceSession("/home/ChatGLM-MNN/onnx/glm_block_0.onnx")

inputs = {'inputs_embeds': inputs_embeds,
          'attention_mask': attention_mask,
          'position_ids': position_ids,
          'past_key_values': past_key_values}

outputs = session.run(None, inputs)

**Comment by Lacacy** [comment]
您好，请问您是如何更改解决该问题的吗？可以赐教一下吗？

**Comment by Lacacy** [comment]
> 参照export.py代码修改了输入的数据shape，可以推理了，看了issue应该是onnx导出的不支持动态shape导致#49

您好，请问您是如何更改解决该问题的吗？可以赐教一下吗？

**Comment by Lacacy** [comment]
@guohuanliang1
